### PR TITLE
Allow boleto orders to conciliate diferent values

### PR DESCRIPTION
**Pull request by bvicenzo**
This PR introduces boleto conciliation with a diferent value of the boleto.
The behavior is:

- When the paid value is lass than the boleto value, The order is not set as paid and the captured amount is the paid value;
- When the paid value is equal to the boleto value, the captured amount is the boleto value and the order is set as boleto paid;
- When the paid value is greater than the boleto value, the captured amount is the paid value and the order is set as boleto paid.

### CODE REVIEW

File: spec/models/order_spec.rb [hunk 1]
```diff
@@ -209,13 +209,40 @@
...
+        expect(order).not_to be_boleto_paid
+
+        order.pay_boleto!('15,00')
+
+        expect(order).not_to be_boleto_paid
+        expect(order.captured_amount).to be < order.amount
+      end
+    end
+
+    context 'when no captured amount is provided' do
```
Comment (j133y) [inline spec/models/order_spec.rb]: s/when no captured amount is provided/when provided captured amount is more than amount/

**Comment by j133y** [comment]
:chic:

File: spec/models/order_spec.rb (hunk 1 above)
Comment (bvicenzo) [inline spec/models/order_spec.rb]: @j133y Fixed here: https://github.com/Baby-com-br/fake-braspag/commit/315b52dcc8e1d57bab7c5b4a55c4786430b13aca
Thanks.

**Comment by antoniofilho** [comment]
:+1:

**Pull request merged**